lib/experimental/html/class-wp-html-walker.php

```diff
@@ -273,25 +273,72 @@ public function next_tag( $query = null ) {
 	/**
 	 * Skips the contents of the title and textarea tags until an appropriate
 	 * tag closer is found:
+	 *
 	 * https://html.spec.whatwg.org/multipage/parsing.html#rcdata-state
 	 *
 	 * @param string $tag_name – the rcdata tag closer to look for.
 	 * @since 6.1.0
 	 */
 	private function skip_rcdata( $tag_name ) {
+		$html       = $this->html;
+		$doc_length = strlen( $html );
+		$tag_length = strlen( $tag_name );
+
+		$at = $this->parsed_bytes;
+
 		while ( true ) {
-			// Find a potential tag closer.
-			$closer = "</{$tag_name}";
-			$at     = strpos( $this->html, $closer, $this->parsed_bytes );
-			if ( false === $at ) {
+			$at = strpos( $this->html, '<', $at );
```

Comment:
This could search for `</` instead of checking for `/` separately

Reply:
oh perfect. good catch